qa/tasks/cephfs/test_volumes.py

```diff
@@ -261,9 +262,7 @@ def test_subvolume_group_create_with_invalid_data_pool_layout(self):
             if ce.exitstatus != errno.EINVAL:
                 raise
         else:
-            raise
-        # clean up
-        self._fs_cmd("subvolumegroup", "rm", self.volname, group, "--force")
```

Comment:
As you're removing this, you'll need to write a separate test that `fs subvolumegroup rm --force` works as expected.